### ISSUE TITLE
NTR: Endoscopy: esophageal mass (Lab: endoscopy and exploratory surgery)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8657]
New term request Endoscopy: esophageal mass (elthat_100321024018): Esophagus  UMLS: 0577008

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000247
- label: Esophageal mass
- parents: [Abnormal esophagus morphology (HP:0002031)](https://hpo.jax.org/app/browse/term/HP:0002031)
- pmid: 
- def: A swelling or enlargment localized to the esophagus. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8657
- synonyms:
